packages/exo/package.json

```diff
@@ -5,7 +5,7 @@
   "keywords": [],
   "author": "Endo contributors",
   "license": "Apache-2.0",
-  "homepage": "https://github.com/endojs/endo/tree/master/packages/exo#readme",
+  "homepage": "https://github.com/endojs/endo/blob/master/packages/exo/README.md",
```

Comment:
Is the [exo README](https://github.com/endojs/endo/blob/master/packages/exo/README.md) intentionally empty?

Reply:
Was not intentional, just not yet written. So I added the text for my recent empty ones: pass-style, patterns, exo. Minimal but much better than nothing. It's a start. Done.